test/parallel/test-child-process-execfile.js

```diff
@@ -59,3 +59,14 @@ const execOpts = { encoding: 'utf8', shell: true };
   execFile(process.execPath, [echoFixture, 0], { signal }, callback);
   ac.abort();
 }
+
+{
+  // Verify that the if something different than Abortcontroller.signal is passed - ERR_INVALID_ARG_TYPE is thrown
```

Comment:
Please keep the capitalization: I'd do something like:
```suggestion
  // Verify that passing a non-signal to execFile throws an ERR_INVALID_ARG_TYPE
```